locales/en/plugin__odf-console.json

```
@@ -1,4 +1,13 @@
 {
+  "Block and File service is unhealthy": "Block and File service is unhealthy",
+  "Object Service is Unhealthy": "Object Service is Unhealthy",
+  "Object Service is unhealthy": "Object Service is unhealthy",
```

```suggestion
  "Object Service is Unhealthy": "Object service is Unhealthy",
  "Object Service is unhealthy": "Object service is unhealthy",
```
Dashboard name is `Object`. So same as in case of `Block and File`, "service" should be lowercase ?